store/listings/management/commands/fetch_listings.py

```diff
@@ -1,44 +1,79 @@
+import logging
+
+from django.conf import settings
 from django.core.management.base import BaseCommand
+from rest_framework import serializers
+
+from store.common.fields import PriceField
+from store.listings.models import Listing
+from store.listings.utils import ListingsRequest
 
-from listings_api_keys import API_KEY, API_TOKEN
-from store.listings.utils import ListingsRequest, save_to_file
+logger = logging.getLogger('listings')
 
 
 class Command(BaseCommand):
-    help = 'Fetch listings from an external API and create text files'
+    help = 'Fetch listings from an external API and create db objects'
 
     def handle(self, *args, **options):
-        api_key = API_KEY
-        api_token = API_TOKEN
-        hostname = 'https://api.pythonic.me/v1'
-
-        listings_request = ListingsRequest(api_key, api_token, hostname)
+        listings_request = ListingsRequest(
+            api_key=settings.PYTHONIC_API_KEY,
+            api_token=settings.PYTHONIC_API_TOKEN,
+            hostname=settings.PYTHONIC_API_HOSTNAME)
 
         listings_data = [
             {
                 'data': listings_request.get_digital_listings(),
-                'filename': 'digital_listings.txt'
             },
             {
                 'data': listings_request.get_soft_tag_listings(),
-                'filename': 'soft_tags.txt'
             },
             {
                 'data': listings_request.get_usd_listings_under_20(),
-                'filename': 'usd_listings_under_20.txt'
             },
             {
                 'data': listings_request.get_blanket_search_listings(),
-                'filename': 'blanket.txt'
             },
             {
                 'data': listings_request.get_cotton_eur_listings(),
-                'filename': 'cotton_eur.txt'
             },
         ]
 
-        for listings in listings_data:
-            save_to_file(listings=listings)
+        from django.db.transaction import atomic
+        with atomic():
+            for listings in listings_data:
+                for listing in listings['data']:
+                    uid = listing.get('listing_id')
+                    if not uid:
+                        logger.error(f"Missing uid for listing: {listing.get('title')}")
+                        continue
+
+                    price = listing.get('price_amount')
+                    if price is None:
+                        logger.error(f"Missing price for listing: {listing.get('title')}")
```

Comment:
Try not to go over 79 chars. [PEP8 standard](https://www.google.com/search?q=PEP8+79+chars)